[2025-10-04T06:42:32.544+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-04T06:42:32.761+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1.docker scheduled__2025-09-29T17:37:00+00:00 [queued]>
[2025-10-04T06:42:32.803+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1.docker scheduled__2025-09-29T17:37:00+00:00 [queued]>
[2025-10-04T06:42:32.805+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-04T06:42:32.892+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): docker> on 2025-09-29 17:37:00+00:00
[2025-10-04T06:42:32.995+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=87) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-04T06:42:33.027+0000] {standard_task_runner.py:63} INFO - Started process 107 to run task
[2025-10-04T06:42:33.037+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'docker', 'scheduled__2025-09-29T17:37:00+00:00', '--job-id', '479', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmpq40d8_wp']
[2025-10-04T06:42:33.056+0000] {standard_task_runner.py:91} INFO - Job 479: Subtask docker
[2025-10-04T06:42:33.335+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-04T06:42:33.656+0000] {task_command.py:426} INFO - Running <TaskInstance: dag1.docker scheduled__2025-09-29T17:37:00+00:00 [running]> on host d0daf19c77b2
[2025-10-04T06:42:34.328+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag1' AIRFLOW_CTX_TASK_ID='docker' AIRFLOW_CTX_EXECUTION_DATE='2025-09-29T17:37:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-29T17:37:00+00:00'
[2025-10-04T06:42:34.334+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-04T06:42:34.336+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-04T06:42:34.344+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'ls -l /tmp']
[2025-10-04T06:42:34.404+0000] {subprocess.py:86} INFO - Output:
[2025-10-04T06:42:34.452+0000] {subprocess.py:93} INFO - total 16
[2025-10-04T06:42:34.454+0000] {subprocess.py:93} INFO - drwx------ 2 root root 4096 Oct  4 06:42 ***tmpalu8riqf
[2025-10-04T06:42:34.456+0000] {subprocess.py:93} INFO - drwx------ 2 root root 4096 Oct  4 06:42 ***tmppfs5is03
[2025-10-04T06:42:34.460+0000] {subprocess.py:93} INFO - drwx------ 2 root root 4096 Oct  4 06:42 ***tmpvj28az2y
[2025-10-04T06:42:34.461+0000] {subprocess.py:93} INFO - drwx------ 2 root root 4096 Oct  4 06:24 pymp-9bou5v9r
[2025-10-04T06:42:34.468+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-04T06:42:34.471+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-04T06:42:34.714+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=docker, run_id=scheduled__2025-09-29T17:37:00+00:00, execution_date=20250929T173700, start_date=20251004T064232, end_date=20251004T064234
[2025-10-04T06:42:34.973+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-04T06:42:35.066+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-04T06:42:35.078+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
